src/main/java/io/confluent/connect/elasticsearch/jest/JestElasticsearchClient.java

```diff
@@ -137,6 +149,19 @@ protected JestElasticsearchClient(Map<String, String> props, JestClientFactory f
             .preemptiveAuthTargetHosts(address.stream()
                 .map(addr -> HttpHost.create(addr)).collect(Collectors.toSet()));
       }
+
+      if (secured) {
+        log.info("Using secured connection");
+        SslFactory kafkaSslFactory = new SslFactory(Mode.CLIENT, "nonev", false);
```

Comment:
is there a typo in `nonev`? I only see `required` / `requested` in the factory's code. Do you mind setting this as a static variable so typos get detected more easily?

Reply:
Great point -- After discussion with @NathanNam, I made this a configurable option since there is enough demand for 2-way in various connectors.